site/content/docs/5.0/guidelines/global-headers.md

```diff
@@ -7,6 +7,19 @@ subgroup: Modules
 toc: true
 ---
 
-{{< callout info >}}
-The feature will be delivered with [#525]({{< param repo >}}/issues/525).
-{{< /callout >}}
+<div class="container">
+  <h2 id="suprabar">Supra bar</h2>
+  <p>
+    <strong>Supra bars are hidden on tablet and mobile devices.</strong>
+    Based on viewport size, supra bar is hidden under <var>768px</var>.
+  </p>
+  <h3 class="mt-5 h5">Supra bar standard — <a id="web-glh-sub-001" class="ui-kit-id">web-glh-sub-001</a></h3>
+</div>
+<nav class="navbar navbar-dark bg-dark navbar-expand-md pt-0 d-none d-sm-block d-sm-none d-md-block">
```

Comment:
The height shouldn't be 40px and the font size 14px?

If so, you could:
- Add `lh-1` class into `<nav>`
- Make the font size 14px

I don't remember if there is a class to have a 14px font size. If not, you will need to add some specific CSS (was done before with the `supra` class).
It could be done directly in this example like this, or by adding a `supra class` only in this example?:

```html
<style>
  .navbar .nav-link {
    font-size: .875rem;
  }
</style>
<nav role="navigation" class="navbar navbar-dark bg-dark navbar-expand-md pt-0 d-none d-sm-block d-sm-none d-md-block lh-1">
  <div class="container">
    <div class="navbar-nav">
      <a href="#" class="nav-link active pb-1" aria-current="page">Personal</a>
      <a class="nav-link pb-1" href="#">Business</a>
    </div>
  </div>
</nav>
```

Reply:
Chose the way you described it, I'm not sure it's a good idea to (re)make supra class. And I'm not sure on how to externalise this css as it's only a snippet of style for an example.

Reply:
If needed in some example, maybe it's possible to extend font-sizes available in bootstrap to add fs-7? 
https://boosted.orange.com/docs/5.0/utilities/text/#font-size take care that only font-size is updated
So if same rendering could be achieve without `.supra` class could be nice. But warning to manage all edge cases (icons, dropdown...) https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/v4-dev/scss/_o-navbar.scss

Reply:
I haven't yet looked at all the specifications but you can do that: https://codepen.io/julien-deramond/pen/dyWzvXM.
If all the design, HTML structure and accessibility rules are correct and as simpler as that, you only need this CSS as you can see in my example:
```css
.fs-7 {
  font-size: .875rem;
}
```
> If needed in some example, maybe it's possible to extend font-sizes available in bootstrap to add fs-7?

I'm not against this proposal

Reply:
fs-7 class has been added